lib/cartopy/_epsg.py

```diff
@@ -41,7 +41,9 @@ class _EPSGProjection(ccrs.Projection):
     def __init__(self, code):
         import pyepsg
         projection = pyepsg.get(code)
-        if not isinstance(projection, pyepsg.ProjectedCRS):
+        if not (isinstance(projection, pyepsg.ProjectedCRS) or
+                (hasattr(pyepsg, 'CompoundCRS') and
```

Comment:
I'd be extremely tempted to update pyepsg dependency to ``>=0.4``. Relevant locations : https://github.com/SciTools/cartopy/blob/master/requirements/epsg.txt and https://github.com/SciTools/cartopy/blob/master/INSTALL

Reply:
Let's go for it.